addon/lib/infinity-model.js

```diff
@@ -4,8 +4,30 @@ import { computed, get, set, getProperties } from '@ember/object';
 import { objectAssign } from '../utils';
 import { typeOf } from '@ember/utils';
 
-
+/**
+  @class InfinityModel
+  @namespace EmberInfinity
+  @module ember-infinity/lib/infinity-model
+  @extends Ember.ArrayProxy
+*/
 export default ArrayProxy.extend({
+  /**
+    @public
+    @property store
+    @default null
+   */
+  store: null,
+
+  /**
+    The supported findMethod name for
+    the developers Ember Data version.
+    Provided here for backwards compat.
+    @public
+    @property storeFindMethod
+    @default null
+   */
+  storeFindMethod: null,
```

Comment:
The model now stores the reference to the store service and find method.  This allows users to specify separate stores/find methods per infinity model per route.  #295

Reply:
👍 good catch